.gitignore

```diff
@@ -0,0 +1 @@
+.DS_Store
```

Comment:
In my laziness on this project i never made a gitignore... I just did that, so hopefully it won't continue to happen. In the meantime, can you manually remove these from the commit?

Reply:
Apparently I'm too tired and can't read... Ignore my comment if you saw it, I missed that you were ADDING a gitignore here :facepalm: 

I also made a gitignore in another branch that's probably more appropriate for this project with some javascripty stuff in it that probably will be used down the road like yarn/node etc, so there will be merge conflicts